packages/web-components/src/badge/badge.ts

```diff
@@ -14,7 +15,7 @@ export class Badge extends FASTBadge {
    * HTML Attribute: appearance
    */
   @attr
-  public appearance: BadgeAppearance = 'filled';
+  public appearance: BadgeProps['appearance'] = 'filled';
```

Comment:
Can you elaborate on coupling these types into a singular typing vs. managing that in react? In react these are all within "props" but that's specifically a react construct.

Reply:
The above is just to illustrate that the typings are the same name/type between the two implementations. How they are implemented was not considered. They could be separate types exported as before, or put into a single object, or some other form.

This was to open the conversation around, "Should we have a single definition that is shared?" Which has moved to #5.